include/GafferUI/StandardNodeGadget.h

```diff
@@ -57,6 +57,7 @@ class NoduleLayout;
 /// - "nodeGadget:minimumWidth" : a node entry with a float value
 /// - "nodeGadget:padding" : a node entry with a float value
 /// - "nodeGadget:color" : Color3f
+/// - "icon" : string naming an image to be used with ImageGadget
```

Comment:
I couldn't figure out why this claims to be an outdated diff... its still called "icon", right?